packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -1307,6 +1307,19 @@ meeting.on('media:stopped', (media) => {
 }
 ```
 
+### Event Caveats
+##### Not seeing remote share if host started sharing screen before participant joins
+
+What apps typically do is call `join()` and then `addMedia()` with `options.mediaSettings.receiveShare=true` 
```

Comment:
I think the context would be easier to follow if it's outlines as a series of steps
```
When trying to receive screen share in an application, standard plugin-meeting workflow is as follow:
1. Set event listener for `media:ready`
2. Call `join()`
3. Call `addMedia()` with `options.mediaSettings.receiveShare=true`
4. Wait to receive an event from `media:ready` with payload `type=remoteShare` that contains the remote share media stream
5. Set `srcObject` of a `video` element in the application to the previously obtained media stream
(e.g. `document.getElementById('remote-screen').srcObject = media.stream`)